2-classes-db/task.md

```diff
@@ -1,79 +1,114 @@
 # Class Database
+
 Run the following commands:
+
 ```
 createdb homework
 psql -d homework -f setup.sql
 ```
+
 This will create and populate a database called homework.
 
 It is based on real data from the government found at
 https://data.gov.uk/dataset/72eaec8e-0d32-4041-a553-87b852abee64/spend-over-25-000-in-worcestershire-acute-hospitals-nhs-trust
 
 You can work with it by running
+
 ```
 psql homework
 ```
+
 ## Submission
 
 Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands after each question.
 
 ### 1. Show the date, transaction_no, description and amount for those transactions whose amount is between £30,000 and £31,000.
+
 ```sql
+SELECT date, transaction_no, description, amount FROM spends WHERE amount>=30000 AND amount<=31000 ;
 
 ```
+
 ### 2. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee'.
-```sql
 
+```sql
+SELECT date, transaction_no, supplier_inv_no,description, amount FROM spends WHERE description LIKE '%fee%';
 ```
+
 ### 3. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'Fee'.
-```sql
 
+```sql
+SELECT date, transaction_no, supplier_inv_no,description, amount FROM spends WHERE description LIKE '%Fee%';
 ```
+
 ### 4. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee' (case insensitive). You will need to search 'https://www.postgresql.org/docs/' to solve this.
-```sql
 
+```sql
+SELECT date, transaction_no, supplier_inv_no,description, amount FROM spends WHERE description ILIKE '%fee%';
 ```
+
 ### 5. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose amount is £25,000, £30,000, £35,000 or £40,000.
-```sql
 
+```sql
+SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE amount IN (25000,30000,35000,40000);
 ```
+
 ### 6. Show the date, the supplier_id, the description and the amount for transactions with the expense area of 'Better Hospital Food'. You could do a query to get the expense_area_id first then do a query to find the dates, supplier_ids and amounts. But it would be better to do this all in one query by linking the tables together using INNER JOINs.
-```sql
 
+```sql
+SELECT spends.date, spends.supplier_id, spends .description, spends .amount FROM spends  INNER JOIN expense_areas  ON spends.expense_area_id = expense_areas.id  WHERE expense_area = 'Better Hospital Food';
```

Comment:
CORRECT - Do be wary, however, of leaving spaces when qualifying fields using a table name (e.g. `spends .description`)  Depending on the parser, this may cause problems.